idaes_examples/notebooks/docs/unit_models/custom_unit_models/creating_unit_model.ipynb

```diff
@@ -218,7 +217,7 @@
     "\n",
     "Creating a State Block requires us to write two classes. The reason we write two classes is because of the inherent nature of how `declare_process_block_data` works. `declare_process_block_data` facilitates creating an `IndexedComponent` object which can handle multiple `ComponentData` objects which represent the component at each point in the indexing set. This makes it easier to build an instance of the model at each indexed point. However, State Blocks are slightly different, as they are always indexed (at least by time). Due to this, we often want to perform actions on all the elements of the indexed StateBlock all at once (rather than element by element).\n",
     "\n",
-    "The class `_StateBlock` is defined without the `declare_process_block_data` decorator and thus works as a traditional class and this facilitates performing a method on the class as a whole rather than individual elements of the indexed property blocks. In this class we define the `release_state` function. `release_states` function is to used to free the state variable from the current state. It takes flags as the argument which determine if the state is to held or can be released. This function uses `revert_state_vars` function which reverts the fixed state of the state variables within an IDAES StateBlock based on a set of flags of the previous state. After releasing the state variables, completion of state release is logged.\n",
+    "The class `_StateBlock` is defined without the `declare_process_block_data` decorator and thus works as a traditional class and this facilitates performing a method on the class as a whole rather than individual elements of the indexed property blocks. In this class we define the `fix_initialization_states` function. `fix_initialization_states` function is to used to fix the state variable within the state block with the provided initial values (usually inlet conditions). It takes a `block` as the argument in which the state variables are to be fixed. It also takes `state_args` as an optional argument. `state_args` is a dictionary with the value for the state variables to be fixed. This function returns a dictionary indexed by the block, state variables and variable index indicating the fixed status of each variable before applying the function. \n",
```

Comment:
You need to use a more specific name than `_StateBlock`, as you define a second version of this later (which will lead to conflicts). Make these `_AqStateBlock`, etc.

Also, I would suggest using full descriptive names for clarity, i.e. `AqueousStateBlock`.